app/models/manageiq/providers/hawkular/middleware_manager/refresh_parser.rb

```diff
@@ -94,6 +95,18 @@ def fetch_server_entities
         end
       end
 
+      def fetch_availability
+        metrics_ids = {}
```

Comment:
Its a hash that maps an already fetched resource with its availability id. (In this case there is Deployments, but could be anything else to fetch them in one call). 
I couldn't think of a good name, maybe `resources_by_availability_id` ? WDYT ?